components/Avatar/avatar.html

```diff
@@ -0,0 +1,290 @@
+              <span class="iconify" data-icon="vaadin:modal-list"></span>
+              <a
+                class="btn-primary-outline-non deco-none"
+                href="./../../components/Modal/modal.html"
+                >Modal</a
+              >
+            </li>
+            <li class="pad-xs list-type-none">
+              <span
+                class="iconify"
+                data-icon="fluent:navigation-16-regular"
+              ></span>
+              <a class="btn-primary-outline-non deco-none" href="#"
+                >Navigation</a
+              >
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="ic:twotone-star-rate"></span>
+              <a class="btn-primary-outline-non deco-none" href="#">Rating</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="bi:grid-1x2"></span>
+              <a
+                class="btn-primary-outline-non deco-none"
+                href="./../../components/Simplified Grid/simplified-grid.html"
+                >Simplified Grid</a
+              >
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="octicon:typography-24"></span>
+              <a
+                class="btn-primary-outline-non deco-none"
+                href="./../../components/Typography/typography.html"
+                >Typography</a
+              >
+            </li>
+          </ul>
+        </div>
+      </aside>
+      <!-- Avatar Main Content starting here -->
+      <div class="main-content content-wrap">
+        <div class="text-5">Avatar</div>
+        <hr class="mar-y-3" />
+        <p class="mar-y-2 text-3">
+          The Avatar component is used to represent a user, and displays the
+          profile picture, initials or fallback icon.
+        </p>
+        <div class="main-content-subblock">
+          <div class="text-4 mar-y-4">Simple Avatar</div>
+
+          <div class="basic_container avt_cont mar-y-2 pad-md">
+            <img
+              class="sm-height xl-bdr-radius"
+              src="./../../assets/avatar.png"
+              height="50"
+              alt=""
+            />
+            <img
+              class="md-height xl-bdr-radius"
+              src="./../../assets/avatar.png"
+              height="50"
+              alt=""
+            />
+            <img
+              class="xl-height xl-bdr-radius"
+              src="./../../assets/avatar.png"
+              height="50"
+              alt=""
+            />
+          </div>
+          <div class="basic_container pad-md mar-y-2">
+            <iframe
+              src="https://carbon.now.sh/embed/t0dPjNUCMqBq3v6nkovf"
+              style="
+                width: 437px;
+                height: 467px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
```

Comment:
this can also be done in css file

Reply:
no that file is code file carbon.sh iframe